overview_tab/map_graph.py

```diff
@@ -4,25 +4,26 @@
 
 FAMILY = "PT Sans"
 
-def data_processing_for_graph(df, postal_code_df):
+def data_processing_for_graph(df, postal_code_df, threshold):
     """
     :param postal_code_df: data frame that includes the postal codes, the latitude and the longitude
-    :param df: agression data
-    :return: a dataframe with one row per postal code as index, with latitude, longitude and number of agressions
+    :param df: survey data
+    :return: a dataframe with one row per postal code as index, with latitude, longitude and number of incidents
```

Comment:
could you add threshold as a param? (Especially because I am not sure what it does)

Reply:
Ok added